course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -332,14 +335,12 @@
             var membersByUserId = {};
             memberList.forEach(function(member) {
                 var memberCopy = angular.copy(member);
-                // this is a temp fix to change the display text of the role
-                // "Teaching Fellow" to "TA" a more perm solution is being
-                // discussed, but will involve talking to the schools.
+
                 if (memberCopy.role && memberCopy.role.role_name == "Teaching Fellow") {
                     memberCopy.role.role_name = "TA";
                 }
                 membersByUserId[memberCopy.user_id] =
-                    (membersByUserId[memberCopy.user_id] || []).push(memberCopy);
+                     (membersByUserId[memberCopy.user_id] || []).concat(memberCopy);
```

Comment:
Yikes, good thing we never needed to actually use these values. Nice catch, I learned something new here.